### ISSUE TITLE
Cement Kiln wasting fuel

### ISSUE THREAD

**Issue by iuriguilherme** [StrangeLoopGames/EcoIssues#23832]
I have a cement kiln using up the coal with work orders that should not trigger that. I suspect it's the ones from the colored reinforced concrete mod. All work orders with vanilla items and modded items have enough materials but not enough labour.

![20230521171103_1](https://github.com/StrangeLoopGames/EcoIssues/assets/2027447/aec52205-faff-468e-8ed3-3bbb1cca241c)
![20230521203521_1](https://github.com/StrangeLoopGames/EcoIssues/assets/2027447/bf6cb638-fc05-4965-ac46-a138573589da)
![20230521203524_1](https://github.com/StrangeLoopGames/EcoIssues/assets/2027447/8ba62c77-ed58-4617-b522-49f46b7b44f4)
![20230521203528_1](https://github.com/StrangeLoopGames/EcoIssues/assets/2027447/871df2c6-5e06-4ac2-a6f7-49938940e787)
![20230521203535_1](https://github.com/StrangeLoopGames/EcoIssues/assets/2027447/2f5f73e9-51a7-42f0-a33d-314df69e26ab)
![20230521203542_1](https://github.com/StrangeLoopGames/EcoIssues/assets/2027447/a8a12e20-2f44-405e-9903-8d2aad10a712)
![20230521203548_1](https://github.com/StrangeLoopGames/EcoIssues/assets/2027447/ed6880fb-7518-4e21-a842-db2c8f398956)
![20230521203600_1](https://github.com/StrangeLoopGames/EcoIssues/assets/2027447/1490c81b-c9e7-4275-a61f-9ffb82a3d236)

**Comment by TwixterZ** [comment]
Hi @iuriguilherme it seems this issue is related to mods used here, have you enquire this to the writer of said mod? 

In any case, can you share what mods is being used when this issue occurs to you?

**Comment by D3nnis3n** [comment]
I'm also not understanding the problem, if it's not mod related can you please explain the problem in more detail? Screenshots are always super helpful, but them alone without any text description can often be even more confusing than no screenshots.

**Comment by iuriguilherme** [comment]
Confirmed it was being caused by colored reinforced concrete (from a mod) work orders. The screenshots are meant to show that all work orders are paused either because of lack of materials or lack of labor in then, but nevertheless the fuel is being consumed. I found another bug with the mod which is that work orders without enough labor will fill the green bar almost to completion but hang on the appropriate iteration. This erroneous behavior was introduced by the mod and therefore shouldn't be addressed here.
